en/guide/vuex-store.md

```diff
@@ -20,24 +20,25 @@ Nuxt.js lets you have **2 modes of store**, choose the one you prefer:
 
 ## Classic mode
 
-To activate the store with the classic mode, we create the `store/index.js` file and export the store instance:
+To activate the store with the classic mode, we create the `store/index.js` file which should export a `createStore` method which returns a Vuex instance:
 
 ```js
 import Vuex from 'vuex'
 
-const store = () => new Vuex.Store({
-
-  state: {
-    counter: 0
-  },
-  mutations: {
-    increment (state) {
-      state.counter++
+const createStore = () => {
```

Comment:
I updated it since it does not have to be named 'createStore' but indeed the name is more explicit.